CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 2.5.2 (Under development)
 
+### App Center Crashes
+
+* **[Fix]** Validate error attachment size to avoid server error or out of memory issues (using the documented limit which is 7MB).
```

Comment:
Maybe 7 "MiB"?

Reply:
As much as I'd like to, I remember a discussion on a docs PR to not use it...

Reply:
Yeah, in public documentation we use terminology `MB` instead of `MiB`.